### ISSUE TITLE
Allow a button to appear in multiple toolbars, and possibly multiple times in the same toolbar

### ISSUE THREAD

**Issue by thekswenson** [xournalpp/xournalpp#3613]
**Is your feature request related to a problem? Please describe.**

I am using the floating toolbox to house most of my pen-related buttons. I would love to have some of those same buttons in the toolbar at the top (e.g. the pen button in both). If I edit `toolbar.ini` and include the same button in multiple places, it only appears in one of those places.

It may also be useful to have the same button in multiple locations on the same toolbar, as I would like to have access to some buttons with either hand while I'm holding the tablet. For example, I often press the save button and would like to use either the stylus in my right hand, or my left pointer finger. Life is easier if I have a save button in each corner.

**Describe the solution you'd like**

Please allow a button to appear as many times as one would like.

**Comment by BertrandSim** [comment]
Indeed, it would be great if I could "copy" a button when customizing the toolbars. A use case for me would be to put frequently used buttons in a floating toolbox, while still having these buttons accessible at the 'main' toolbars at the top and sides. Thanks!

**Comment by BertrandSim** [comment]
Indeed, it would be great if I could "copy" a button when customizing the toolbars. A use case for me would be to put frequently used buttons in a floating toolbox, while still having these buttons accessible at the 'main' toolbars at the top and sides. Thanks!

**Comment by atticus-sullivan** [comment]
Oh yes this feature is very much needed to really start working with the *floating toolbox*.

For a quick and dirty solution, one might make use of https://github.com/xournalpp/xournalpp/pull/4671, re-implement the button as plugin and create button for it. But still this is only a (very dirty) workaround and no replacement of the real feature.
